src/transactions/TransactionFrameBase.cpp

```diff
@@ -25,4 +25,4 @@ TransactionFrameBase::makeTransactionFromWire(Hash const& networkID,
     }
 }
 
-}
\ No newline at end of file
+}
```

Comment:
Nit: newline

Reply:
I think it's fine? We usually have new lines at the end of files; this probably has been introduced by automatic formatting at some point.